atest/testdata/standard_libraries/collections/dictionary.robot

```diff
@@ -118,7 +118,7 @@ Dictionaries Should Be Equal
     Dictionaries Should Be Equal    ${BIG}    ${BIG}
 
 Dictionaries Of Different Type Should Be Equal
-    ${big2}=    Evaluate    robot.utils.OrderedDict($BIG)    modules=robot
+    ${big2}=    Evaluate    collections.OrderedDict($BIG)    modules=robot
```

Comment:
Hang on, I spoke a bit too soon... This and the next change is failing on Python 2.7. How should `collections ` be imported?

https://travis-ci.org/hugovk/robotframework/jobs/287886163#L524

(Lucky I did enable and try this out on the CI!)

Reply:
You can change the second argument to `modules=collections`.

Reply:
Thanks, that now passes in the parallel CI branch:

https://travis-ci.org/hugovk/robotframework/jobs/288976478